tests/ci/stratisd.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+set -e
+
+if [ -z "$WORKSPACE" ]
+then
+    echo "Required WORKSPACE environment variable not set"
+    exit 1
+fi
+
+export PATH="~/.cargo/bin:$PATH"
+export STRATIS_DEPS_DIR=$WORKSPACE/stratis-deps
+export RUST_LOG=libstratis=debug
+export TEST_BLOCKDEVS_FILE=~/test_config.json
+
+TARGET=$1
+
+if [ -z $TARGET ]
+then
+   echo "Usage: $0 test-loop | test-real"
+   exit 1
+fi
+
+# Jenkins will set WORKSPACE to the top level directory that contains
+# stratisd  git repo
+if [ -z $WORKSPACE ]
```

Comment:
Why is WORKSPACE variable checked for twice? Maybe you can omit the first check and just fix up spelling here?

Reply:
Ok.